regexes/device/mobiles.yml

```diff
@@ -3067,6 +3067,12 @@ Lenovo:
     - regex: 'Yoga Tablet([^;/]+) Build'
       model: 'Yoga Tablet$1'
       device: 'tablet'
+    - regex: 'Lenovo TB\-X704F Build/NMF26F'
+      model: 'Lenovo Tab 4 10 Plus'
```

Comment:
would you remove the brand name from the model name? The model should only be `Tab 4 10 Plus`.

Reply:
\+